widgets/features/features.php

```diff
@@ -8,6 +8,11 @@
 
 class SiteOrigin_Widget_Features_Widget extends SiteOrigin_Widget {
 	function __construct() {
+		$image_size_configs = $this->get_image_sizes();
```

Comment:
Generally for this heavy code stuff, we should be implementing form options in the `modify_form` function. Huge advantage there is that this function is only called when the form is generated, while the constructor is called on every page load.

One day I'd actually like to move all form options out of the constructors. To make things a little more efficient.